index.js

```diff
@@ -5,6 +5,8 @@
  */
 'use strict';
 var http = require('http');
+
+// Deprecated Libraries
```

Comment:
Please use full message that I suggested, pointing people to the readme for latest info.